front/src/Components/CreateProfile.jsx

```diff
@@ -8,19 +8,19 @@ const CreateProfile = () => {
             <br />
             <input type='text' name='nickname' id='nickname' required />
             <br />
-            <label htmlFor='firstname'>Prénom :</label>
+            <label htmlFor='email'>Adresse mail :</label>
             <br />
```

Comment:
don't use `<br>` for design: use css instead. e.g. add a `display: block` to your labels and inputs